scalyr_agent/scalyr_client.py

```diff
@@ -633,35 +633,47 @@ def __receive_response(self, body_str, send_time):
                     response = '{ "status":"success" }'
                     status_code = 200
                 else:
+                    start_ts = time.time()
                     status_code = self.__connection.status_code()
                     response = self.__connection.response()
+                    end_ts = time.time()
```

Comment:
It seems that this end timestamp ends up being unused if no exception is occurred.